tasks/validate_host_names.yaml

```diff
@@ -1,24 +1,10 @@
 - fail:
-    msg: "Please revise your vars.yaml file invalid characters in hostnames"
+    msg: "Please revise your vars.yaml file. Invalid characters found in hostnames"
   when: item is search('{{ chars }}')
   with_items:
   - "{{ dns.domain }}"
   - "{{ helper.name }}"
   - "{{ bootstrap.name }}"
-  register: val0
-  ignore_errors: True
-
-- fail:
-    msg: "Please revise your vars.yaml file invalid characters in hostnames"
-  when: item.name is search('{{ chars }}')
-  with_items:
   - "{{ masters }}"
-  - "{{ workers }}"
-  register: val1
-  ignore_errors: True
-
-- fail:
-    msg: "Please revise your vars.yaml file invalid characters in hostnames"
-  when:
-    (val0.failed is defined) or
-    (val1.failed is defined)
+  - "{{ workers | default('') }}"
+  register: val0
```

Comment:
i think you can drop the result of this task. i.e. no `register`

Reply:
Yup, you're right not needed. Pushed update.